smart-contracts/contracts/mixins/MixinPurchase.sol

```diff
@@ -404,8 +384,6 @@ contract MixinPurchase is
     address[] memory _keyManagers,
     bytes[] calldata _data
   ) external payable returns (uint[] memory) {
-    _lockPurchaseIsPossible(_recipients.length);
```

Comment:
Are we doing this in the Struct-purchase function?

Reply:
yes